python/tests/server/test_http_input.py

```diff
@@ -216,18 +216,18 @@ def predict(
 
 def test_gt_lt():
     class Predictor(BasePredictor):
-        def predict(self, num: float = Input(gt=3, lt=10.5)) -> float:
+        def predict(self, num: float = Input(ge=3.01, le=10.5)) -> float:
```

Comment:
This could still just be `3`, right?

Reply:
For some reason I thought it was important to maintain the test's original restrictions on having the value be greater-than-but-not-equal-to 3. But then after I had done all that I noticed that the actual test value that's passed through is 5 so.... yeah, it just remain 3. Whatever you wanna do